node/pkg/watchers/evm/watcher.go

```diff
@@ -882,8 +882,16 @@ func (w *Watcher) verifyAndPublish(
 			return err
 		}
 		msg = &verifiedMsg
+		w.logger.Info(
+			"verified transfer",
+			msg.ZapFields()...,
+		)
 	}
 
+	w.logger.Info(
```

Comment:
Not sure I like that we will now be logging two or three messages for every observation (the original "message observed" plus these two). Also, bear in mind that by default info logging in disabled in CI so you won't see these anyway. But we will see them in prod.

Reply:
I mostly had these in mind for local testing and testnet. I think they're warranted because they're reporting different parts of the process (initial observation, evaluation, and actual publishing). I'd be happy to change these to Debug level so that they're filtered out of prod. Would that work?